docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -70,6 +73,21 @@ The data nodes that are part of a **scenario**, **pipeline** or **task** can be
     task.sales_history
     ```
 
+Data nodes can be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^`, which will return
+a list of existing data nodes based on the config_id.
+
+!!! Example
+
+    ```python linenums="1"
+    import taipy as tp
+    import my_config
+
+    scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
```

Comment:
Maybe we can instantiate 2 scenarios in the example and explain that calling `tp.get_entities_by_config_id("trained_model")` returns 2 data nodes.